<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tailor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>Sheet</string>
    </property>
    <addaction name="actionAdd_Data_Sheet"/>
    <addaction name="actionDuplicate_Data_Sheet"/>
    <addaction name="actionDuplicate_Data_Sheet_With_Plots"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_column"/>
    <addaction name="actionAdd_calculated_column"/>
    <addaction name="actionAdd_row"/>
    <addaction name="separator"/>
    <addaction name="actionRemove_column"/>
    <addaction name="actionRemove_row"/>
    <addaction name="actionClear_Cell_Contents"/>
    <addaction name="separator"/>
    <addaction name="actionRename_Data_Sheet"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
     <addaction name="actionClear_Menu"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport_CSV"/>
    <addaction name="actionExport_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionPreview_Graph"/>
    <addaction name="actionExport_Graph_to_PDF"/>
    <addaction name="actionExport_Graph_to_PNG"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Tailor"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionCreate_Plot"/>
    <addaction name="actionDuplicate_Plot"/>
    <addaction name="actionCreate_MultiPlot"/>
    <addaction name="actionChange_Plot_Source"/>
    <addaction name="separator"/>
    <addaction name="actionRename_Plot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTable"/>
   <addaction name="menuPlot"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_column">
   <property name="text">
    <string>Add Column</string>
   </property>
  </action>
  <action name="actionAdd_calculated_column">
   <property name="text">
    <string>Add Calculated Column</string>
   </property>
  </action>
  <action name="actionAdd_row">
   <property name="text">
    <string>Add Row</string>
   </property>
  </action>
  <action name="actionRemove_column">
   <property name="text">
    <string>Remove Columns</string>
   </property>
  </action>
  <action name="actionRemove_row">
   <property name="text">
    <string>Remove Rows</string>
   </property>
  </action>
  <action name="actionImport_CSV">
   <property name="text">
    <string>Import CSV</string>
   </property>
  </action>
  <action name="actionExport_CSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
  <action name="actionExport_Graph_to_PDF">
   <property name="text">
    <string>Export Graph to PDF</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExport_Graph_to_PNG">
   <property name="text">
    <string>Export Graph to PNG</string>
   </property>
  </action>
  <action name="actionType_Strange_Things">
   <property name="text">
    <string>Type Strange Things</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Tailor">
   <property name="text">
    <string>About Tailor</string>
   </property>
  </action>
  <action name="actionImport_CSV_into_project">
   <property name="text">
    <string>Import CSV Into Current Project</string>
   </property>
  </action>
  <action name="actionImport_CSV_Into_Current_Project">
   <property name="text">
    <string>Import CSV Into Current Project</string>
   </property>
  </action>
  <action name="actionClear_Cell_Contents">
   <property name="text">
    <string>Clear Cell Contents</string>
   </property>
  </action>
  <action name="actionClear_Menu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear Menu</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionAdd_Data_Sheet">
   <property name="text">
    <string>Add Data Sheet</string>
   </property>
  </action>
  <action name="actionDuplicate_Data_Sheet">
   <property name="text">
    <string>Duplicate Data Sheet</string>
   </property>
  </action>
  <action name="actionDuplicate_Plot">
   <property name="text">
    <string>Duplicate Plot</string>
   </property>
  </action>
  <action name="actionDuplicate_Data_Sheet_With_Plots">
   <property name="text">
    <string>Duplicate Data Sheet With Plots</string>
   </property>
  </action>
  <action name="actionChange_Plot_Source">
   <property name="text">
    <string>Change Plot Source</string>
   </property>
  </action>
  <action name="actionRename_Data_Sheet">
   <property name="text">
    <string>Rename Data Sheet</string>
   </property>
  </action>
  <action name="actionRename_Plot">
   <property name="text">
    <string>Rename Plot</string>
   </property>
  </action>
  <action name="actionPreview_Graph">
   <property name="text">
    <string>Preview Graph</string>
   </property>
  </action>
  <action name="actionCreate_MultiPlot">
   <property name="text">
    <string>Create MultiPlot</string>
   </property>
  </action>
  <action name="actionCreate_Plot">
   <property name="text">
    <string>Create Plot</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
